### ISSUE TITLE
Can you make a 2D Drawing (PDF) to 3D Model (DWG) file?

### ISSUE THREAD

**Issue by Mohdjariullah** [Brock123/3D-Model-to-Pdf-Conversion#1]

**Comment by amit-k-pathak** [comment]
No, this project is all about converting DWG files to 2D pdf only, not the
reverse.

**Comment by Mohdjariullah** [comment]
Can you please make it? Make a separate project:)

**Comment by amit-k-pathak** [comment]
Currently I do not possess the expertise for the reverse conversion. Also
DWG is a proprietary file format so it's bit tricky to generate one. You
can explore other alternatives such as
https://www.investintech.com/pdf-to-dwg/.
